test/Test/Stan/Toml.hs

```diff
@@ -24,9 +25,9 @@ tomlSpec = describe "TOML configuration tests" $ do
     it "Parses test/.stan.example.toml" $
         Toml.decodeFile configCodec "test/.stan-example.toml" `shouldReturn` configExample
   where
-    configExample :: Config
-    configExample = Config
-        { configChecks =
+    configExample :: PartialConfig
+    configExample = ConfigP
+        { configChecks = ("TOML",) <$> pure
```

Comment:
Does it look batter with the `withTag` function? Or do we need `pureTagged` in `trial`? :slightly_smiling_face: 
```suggestion
        { configChecks = withTag "TOML" $ pure
```